model/group_impl.go

```diff
@@ -33,6 +35,44 @@ func (repo *EntRepository) CreateGroup(ctx context.Context, name string, descrip
 	return ConvertEntGroupToModelGroup(created), nil
 }
 
+func (repo *EntRepository) GetMembers(ctx context.Context, groupID uuid.UUID) ([]*User, error) {
+	members, err := repo.client.Group.
+	Query().
+	Where(group.IDEQ(groupID)).
+	QueryUser().
+	All(ctx)
```

Comment:
```suggestion
	members, err := repo.client.Group.
	    Query().
	    Where(group.IDEQ(groupID)).
	    QueryUser().
	    All(ctx)
```
こんな感じでインデントを入れてほしいです。ほかのとこもお願いします。